json-rpc/src/tests/unit_tests.rs

```diff
@@ -189,6 +189,7 @@ fn test_json_rpc_protocol() {
     let data: JsonMap = resp.json().unwrap();
     assert!(data.get(JSONRPC_LIBRA_LEDGER_VERSION).is_some());
     assert!(data.get(JSONRPC_LIBRA_LEDGER_TIMESTAMPUSECS).is_some());
+    assert!(data.get(JSONRPC_LIBRA_CHAIN_ID).is_some());
```

Comment:
can we go further and check that chain_id == ChainId::test?

Reply:
done.